### ISSUE TITLE
Me: 2fa: add title to print page for 2fa backup codes

### ISSUE THREAD

**Issue by v18** [Automattic/wp-calypso#398]
When printing backup codes in Chrome the default title for the document is "Untitled". Which is no big deal if you're printing directly, but I was saving mine temporarily to print them all in a batch and then delete. It would be a simple change to have it be something generic like "WordPress.com - Backup Codes for usernamehere"

Steps to reproduce:
1. Go to /me/security
2. Generate new backup codes
3. Print them
4. See how the page doesn't have a title?

<img width="384" alt="screen shot 2015-07-14 at 23 07 42" src="https://cloud.githubusercontent.com/assets/3073557/8685242/a5a38aca-2a7d-11e5-812f-2f3c8e52cd5a.png">

<img width="1392" alt="a9a21b6e-2a7d-11e5-971c-d241d423f79a" src="https://cloud.githubusercontent.com/assets/11487924/11320928/6ac318f0-9079-11e5-8fc1-e423fd77cfd7.png">

This is related to #385

**Comment by noxee** [comment]
Is this still need?

In `client/me/security-2fa-backup-codes-list/index.jsx` there is now a title being set as shown in the code below:

```javascript
let html = '<html><head><title>';

html += this.translate( 'Backup verification codes' );
html += '</title></head>';
```

**Comment by v18** [comment]
@noxee - were you able to see a title when printing the 2FA codes from the site?

**Comment by noxee** [comment]
@v18 I can see title **Backup verification codes** as shown in the screenshot below.

![screen shot 2016-04-13 at 9 53 34 am](https://cloud.githubusercontent.com/assets/1409676/14500077/e4cb672e-015d-11e6-9f84-3376a21aa29c.png)

**Comment by blowery** [comment]
Closing this as good enough. I don't think we want our domain name on the set of code.

**Comment by josephscott** [comment]
@blowery I'm going to disagree.  I recently ran into this exact issue. I've got a significant stack of paper printouts of 2fa backup codes for various services that I use.  If I hadn't noticed the lack of detail WordPress.com provided and wrote down the details with pen, these backup codes would be essentially useless.

The nearly blank list that WordPress.com provides is a problem waiting to happen, at the worst possible moment.

At a minimum the site name ( WordPress.com ) should be included in the print out.  Second, and really for me just as important, the username associated with the account should be shown as well.  

For that second item, we already do that for the [file name when saving](https://github.com/Automattic/wp-calypso/blob/master/client/me/security-2fa-backup-codes-list/index.jsx#L111).  The printout should get the same benefit.  For people managing multiple WordPress.com accounts this is a big deal.

The username should already be available to [getBackupCodeHTML() ](https://github.com/Automattic/wp-calypso/blob/master/client/me/security-2fa-backup-codes-list/index.jsx#L144) via `this.props`, so I don't expect that would be hard to add.  The site name might be a bit more involved, if we want it for use outside of WordPress.com ( is the site name a config level option some where? ).

/cc @sgomes
